manifest-build-tools/post_test.sh

```diff
@@ -0,0 +1,217 @@
+#!/bin/bash
+
+############################################
+# Post-Test scripts for ova, vagrantBox and docker.
+# This script will up/deploy ova, box and docker images,
+# Then check if RackHD services are good running.
+#
+# Usage: 
+# post_test.sh \
+# --type ova, vagrant or docker
+#
+# ova-post-test need some special parameter of net config and target esxi server
+# --adminIP ***.***.***.***
+# --adminGateway ***.***.***.***
+# --adminNetmask 255.255.255.0
+# --adminDNS ***.***.***.***
+# --datastore some-datastore
+# --deployName ova-for-post-test
+# --ovaFile /someDir/some.ova
+# --vcenterHost ***.***.***.***
+# --ntName user
+# --ntPass password
+# --esxiHost ***.***.***.***
+#
+# vagrant-post-test need some special parameter of boxFile and controlNetwork name
+# --boxFile ./someDir/some.box
+# --controlNetwork vmnet*
+#
+# docker-post-test need some special parameter of docker build record file and cloned RackHD repo
+# --RackHDDir ./someDir/RackHD
+# --buildRecord ./record_file
+#
+############################################
+
+set -x
+
+while [ "$1" != "" ];do
+    case $1 in
+        --type)
+            shift
+            type=$1;;
+        --adminIP)
+            shift
+            adminIP=$1;;
+        --adminGateway)
+            shift
+            adminGateway=$1;;
+        --adminNetmask)
+            shift
+            adminNetmask=$1;;
+        --adminDNS)
+            shift
+            adminDNS=$1;;
+        --datastore)
+            shift
+            datastore=$1;;
+        --deployName)
+            shift
+            deployName=$1;;
+        --ovaFile)
+            shift
+            ovaFile=$1;;
+        --vcenterHost)
+            shift
+            vcenterHost=$1;;
+        --ntName)
+            shift
+            ntName=$1;;
+        --ntPass)
+            shift
+            ntPass=$1;;
+        --esxiHost)
+            shift
+            esxiHost=$1;;
+        --boxFile)
+            shift
+            boxFile=$1;;
+        --controlNetwork)
+            shift
+            controlNetwork=$1;;
+        --RackHDDir)
+            shift
+            RackHDDir=$1;;
+        --buildRecord)
+            shift
+            buildRecord=$1;;
+        *)
+        exit 1
+    esac
+    shift
+done
+
+findRackHDService() {
+    case $type in
+        ova)
+        api_test_result=`ansible ova-for-post-test -a "wget --retry-connrefused --waitretry=1 --read-timeout=20 --timeout=15 -t 1 --continue localhost:8080/api/2.0/nodes"`
+        echo $api_test_result | grep "$service_normal_sentence" > /dev/null  2>&1
+        ;;
+        docker)
+        wget --retry-connrefused --waitretry=1 --read-timeout=20 --timeout=15 -t 1 --continue http://172.31.128.1:9080/api/2.0/nodes
+        ;;
+        vagrant)
+        curl 127.0.0.1:9090/api/2.0/nodes
```

Comment:
can you comment where the 9090 comes from.
in case in the future , the port changed, but maintainer will have no idea where this 9090 originated.